src/zos_files/zowe/zos_files_for_zowe_sdk/files.py

```diff
@@ -68,6 +71,24 @@ def get_file_content(self, filepath_name):
         response_json = self.request_handler.perform_request("GET", custom_args)
         return response_json
 
+    def delete_uss(self, filepath_name):
+        """
+        Delete a file or directory
+
+        Parameters
+        ----------
+        filepath of the file to be deleted
+
+        Returns
+        -------
+        204
+            HTTP Response for No Content
+        """
+        custom_args = self.create_custom_request_arguments()
+        custom_args["url"] = "{}fs{}".format(self.request_endpoint,filepath_name)
+        response_json = self.request_handler.perform_request("DELETE", custom_args)
```

Comment:
Thanks for documenting the expected response code of 204 👍 It also needs to be specified on the `perform_request` method with the argument `expected_code=[204]` in order for the request to succeed with a non-200 response code.

Reply:
Yaa, thanks for the review. I have made the necessary changes to the code.